### PR TITLE
Dalia Guevara - WDPT - abril2023

### PR DESCRIPTION
**Pull request by DaliaElena**

### CODE REVIEW

**Comment by DaliaElena** [comment]
> _ No description provided. _

Dalia Guevara - WDPT - abril2023

File: src/clue.js [hunk 1]
```diff
@@ -2,25 +2,148 @@
...
+
+function selectRandom(todo_lo_que_yo_te_mande) {
+    let array_length = todo_lo_que_yo_te_mande.length ;
+    let random_number = Math.floor(Math.random() * array_length);
+    let element_ranndom = todo_lo_que_yo_te_mande[random_number];
+    return element_ranndom
+
+}
+
+// console.log(selectRandom(suspectsArray))
```
Comment (ta-web-mexpt2) [inline src/clue.js:115]: try not lo leave console on your code is good practice eliminate those

File: src/clue.js [hunk 2]
```diff
@@ -2,25 +2,148 @@
...
 
+}
+// console.log(pickMystery())
 
 // ITERATION 3
 
-function revealMystery() {}
+function revealMystery(envelope) {
+    // let killer = pickMystery()
+    let killer = envelope
```
Comment (ta-web-mexpt2) [inline src/clue.js:135]: god use of the variables for better read , but when is only a return string is better using dot notation

**Review by ta-web-mexpt2 — COMMENTED** [review]
great work in general just have in mind those changes

**Pull request closed without merging**